mysite/deployment_settings.py

```diff
@@ -50,7 +48,10 @@
 # svn mission requires a subdomain svn-mission for cloudflare to
 # work properly
 SVN_REPO_URL_PREFIX = 'svn://svn-mission.openhatch.org/'
-GIT_REPO_URL_PREFIX = 'https://openhatch.org/git-mission-data/git/'
+GIT_REPO_URL_PREFIX = 'https://git-mission.openhatch.org/git-mission-data/git/'
```

Comment:
I've just GPG emailed you the CloudFlare password, which should be all the
credentials you need to configure DNS. Let me know if you agree/disagree!